app/domain/authentication/installed_authenticators.rb

```diff
@@ -33,21 +33,18 @@ def configured_authenticators
           .push(::Authentication::Common.default_authenticator_name)
       end
 
-      def enabled_authenticators(env)
+      def enabled_authenticators
         # Enabling via environment overrides enabling via CLI
-        env_enabled_authenticators(env) || db_enabled_authenticators
+        authenticators = 
+          Rails.application.config.conjur_config.authenticators.uniq
```

Comment:
What do you think about moving the `uniq` call to the `authenticators` method in the config?  (We may need to create a new method that extends the parent class behaviour).  To me, it feels out of place here to "fix-up" config.

Reply:
We might also want to strip empty/nil items from the array.